exercises/02-publish-subscribe/after/Divergent.Customers/Divergent.Customers.csproj

```diff
@@ -7,6 +7,9 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="NServiceBus" Version="7.0.1" />
+    <PackageReference Include="NServiceBus.Persistence.Sql" Version="4.2.0" />
+    <PackageReference Include="NServiceBus.Persistence.Sql.MsBuild" Version="4.2.0" />
```

Comment:
These changes should not be made here. It is the shared `ITOps.EndpointConfiguration' project that defines the transport. Change should be made there: https://github.com/Particular/Workshop/blob/6aee55e19b8d85f431f76eaa9971176f2c2eff63/exercises/shared-api-gateway/ITOps.EndpointConfig/ITOps.EndpointConfig.csproj#L8

The same applies to all the other places where these references have been added.